packages/cw-orch-mock/src/core.rs

```diff
@@ -576,8 +614,8 @@ mod test {
     fn bank_querier_works() -> Result<(), CwEnvError> {
         let denom = "urandom";
         let init_coins = coins(45, denom);
-        let sender = Addr::unchecked("sender");
-        let app = Mock::new(&sender);
+        let app = Mock::new("sender");
+        let sender = app.addr_make("sender");
```

Comment:
Almost every test has these two lines. Maybe `Mock::new` should return `(Mock, Addr)`?

Reply:
Don't really like that, when we create a mock, we should get the app only.
If the sender is needed, they can query via `app.sender()` which returns a valid Addr

Reply:
actually not really, maybe `Mock::new` can take `&Addr` and a new `Mock::new_str` can take `impl Into<String>` and return a tuple?